include/sound/soc-acpi.h

```diff
@@ -59,15 +59,17 @@ static inline struct snd_soc_acpi_mach *snd_soc_acpi_codec_list(void *arg)
  *
  * @acpi_ipc_irq_index: used for BYT-CR detection
  * @platform: string used for HDaudio codec support
- * @codec_mask: used for HDAudio support
+ * @hda_codec_mask: used for HDAudio support
+ * @I2S_codec_mask: used for I2S codec support
```

Comment:
I don't really see why this is needed? link_mask has a clear definition for SoundWire. codec_mask is also well defined for HDaudio. I don't see at all what this would mean for I2S.
What are you trying to do with this? Is this to represent on which SSP a codec is connected, or which SSPs are used in a given platform?

Reply:
done